salt/renderers/jinja.py

```diff
@@ -293,7 +294,7 @@ def _split_module_dicts():
 
         {{ salt.cmd.run('uptime') }}
     '''
-    if not isinstance(__salt__, dict):
+    if not isinstance(__salt__, (dict, salt.loader.LazyLoader)):
         return __salt__
     mod_dict = dict(__salt__)
```

Comment:
In my tests this line breaks the mod_dict, the lazy loader does not return a dict of funcs